geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala

```diff
@@ -151,7 +151,12 @@ class SpatialTiledRasterLayer(
       rdd.metadata
     )
 
-    val _neighborhood = getNeighborhood(operation, neighborhood, param1, param2, param3)
+    val _neighborhood =
+      if (operation == Constants.SLOPE || operation == Constants.ASPECT)
+        getNeighborhood(operation, neighborhood, 1.0, 0.0, 0.0)
```

Comment:
I know you didn't create the method, but looking at the [getNeighborhood](https://github.com/locationtech-labs/geopyspark/blob/fe5588d06ee3137705102a6f2af3ff6412b4280b/geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/GeoTrellisUtils.scala#L50) method, it looks like that `operation` parameter is not used at all and can be removed.